py/client-ticking/build.gradle

```diff
@@ -105,95 +97,83 @@ def testPyClientTickingManyLinux = { wheelsSet, taskName, parentContainer, image
-                        pip install --upgrade pip; \
-                        pip install unittest-xml-reporting; \
-                        deactivate; \
-                      done; \
-                      $DNF clean all; \
-                      rm -fr /out; \
-                      mkdir -p \
-                        /out/report \
-                        /project/tests                      
+        runCommand(
+                '''set -eux; \
+                   DNF=`type microdnf >/dev/null 2>&1 && echo 'microdnf --disableplugin=subscription-manager' || echo 'dnf -q'`; \
+                   $DNF -y update
                    ''')
-        copyFile('tests/', "/project/tests/")
-        copyFile('dep-wheels/', "/project/dep-wheels")
-        copyFile('pyt-wheels/', "/project/pyt-wheels")
-        workingDir('/project')
-        runCommand("WHEELS_SET='${wheelsSet}'; " +
-                   '''set -eux ; \
-                      for spec in ${WHEELS_SET}; do \
-                        pyver=`echo "$spec" | cut -d: -f 1`; \
-                        tag=`echo "$spec" | cut -d: -f 2`; \
-                        [ -f /project/pyt-wheels/pydeephaven_ticking*"${tag}"*.whl ]; \
-                        source "/project/$pyver/bin/activate"; \
-                        pip install unittest-xml-reporting; \
-                        pip install /project/dep-wheels/*.whl; \
-                        pip install /project/pyt-wheels/pydeephaven_ticking*"${tag}"*.whl; \
-                        deactivate; \
-                     done
+        runCommand("PYTHON_VERSION='${pythonVersion}'; " +
+                '''set -eux; \
+                   DNF=`type microdnf >/dev/null 2>&1 && echo 'microdnf --disableplugin=subscription-manager' || echo 'dnf -q'`; \
+                   $DNF -y install "python${PYTHON_VERSION}"; \
+                   "python${PYTHON_VERSION}" -m venv "/project/${PYTHON_VERSION}"; \
+                   source "/project/${PYTHON_VERSION}/bin/activate"; \
+                   pip install --upgrade pip; \
+                   pip install unittest-xml-reporting; \
+                   rm -fr /out; \
+                   mkdir -p \
+                       /out/report \
+                       /project/tests
                    ''')
-
+        copyFile('tests/', '/project/tests/')
+        copyFile('dep-wheels/', '/project/dep-wheels')
+        copyFile('pyt-wheels/', '/project/pyt-wheels')
+        copyFile('testPyClientTicking/entrypoint.sh', '/entrypoint.sh')
+        workingDir('/project')
         //
         // Setup for test run.
         //
-        environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
-        environmentVariable 'DH_PORT', '10000'
+        environmentVariable([
+                DH_HOST: deephavenDocker.containerName.get(),
+                DH_PORT: '10000',
+                PYTHON_VERSION: pythonVersion
+        ])
+        entrypoint = ['/entrypoint.sh']
```

Comment:
Why define this here at all, rather than the commented out one below?

Reply:
We determined my usage actually _was_ setting it on the outer side (the inner dockerfile calls it `entryPoint`). I've moved it.